packages/web/components/Modal/__tests__/BeAReviewerModal.js

```diff
@@ -39,6 +39,8 @@ describe('<BeAReviewerModal />', () => {
 	});
 
 	it('should be able to add category and sub-category to selected list and remove it', async () => {
+		jest.setTimeout(30000);
```

Comment:
Had to add this custom timeout because this test always breaks in my computer.

Low-end device  👎🏻

Maybe we should refactor this in a later moment